src/main/java/com/cburch/logisim/std/wiring/Pin.java

```diff
@@ -440,6 +587,10 @@ public void mouseReleased(InstanceState state, MouseEvent e) {
         EditDecimal dialog = new EditDecimal(state);
         dialog.setLocation(e.getXOnScreen() - 60, e.getYOnScreen() - 40);
         dialog.setVisible(true);
+      } else if (radix == RadixOption.RADIX_FLOAT) {
+        EditFloat dialog = new EditFloat(state);
```

Comment:
I know this class needs `final var` in other places too, but make your dialog `final var`